### PR TITLE
Update binary-release.md

### PR DESCRIPTION
**Pull request by hnjolles1**
https://github.com/livepeer/docs/issues/160

### CODE REVIEW

File: docs/installation/install-livepeer/binary-release.md [hunk 1]
```diff
@@ -40,19 +39,37 @@ mv livepeer-linux-amd64/* /usr/local/bin/
...
+```shell
+# <RELEASE_VERSION> is the release version, e.g. v0.5.29
 
-# First, fetch the latest release
+# Fetch the latest release .tar.gz 
 wget https://github.com/livepeer/go-livepeer/releases/download/<RELEASE_VERSION>/livepeer-windows-amd64.tar.gz
 
 # Next, extract it
-tar -zxvf livepeer-windows-amd64.tar.gz
+tar -zxvf li2
```
Comment (leszko) [inline docs/installation/install-livepeer/binary-release.md:51]: What is this `li2`? Should we keep `livepeer-windows-amd64.tar.gz`?

File: docs/installation/install-livepeer/binary-release.md [hunk 2]
```diff
@@ -40,19 +39,37 @@ mv livepeer-linux-amd64/* /usr/local/bin/
...
 
-# Finally, move it to the appropriate directory
-mv livepeer-windows-amd64/* /usr/local/bin/
+# Finally, move it to the appropriate directory, e.g. C:/Users/usr/Desktop/livepeer-folder
+move livepeer-windows-amd64 /*/usr/local/<folder-name>
```
Comment (leszko) [inline docs/installation/install-livepeer/binary-release.md:54]: If it's for Windows, shouldn't we just have it like this?

```
move livepeer-windows-amd64 C:/Users/usr/Desktop/livepeer-folder/
```

File: docs/installation/install-livepeer/binary-release.md (hunk 1 above)
Comment (hnjolles1) [reply docs/installation/install-livepeer/binary-release.md:51]: no idea, that was on the version i had, maybe i changed it by accident. Thanks for pointing it out.

File: docs/installation/install-livepeer/binary-release.md (hunk 2 above)
Comment (hnjolles1) [reply docs/installation/install-livepeer/binary-release.md:54]: well, they can save it where ever they want, we aren't prescribing, that's why we made an e.g.

File: docs/installation/install-livepeer/binary-release.md (hunk 1 above)
Comment (leszko) [reply docs/installation/install-livepeer/binary-release.md:51]: yeah, I'd change it to `livepeer-windows-amd64.tar.gz`, `li2` seems like a mistake.

File: docs/installation/install-livepeer/binary-release.md (hunk 2 above)
Comment (leszko) [reply docs/installation/install-livepeer/binary-release.md:54]: Yes, but this description is dedicated to Windows. The `move` command is only for Windows. And in Windows, you don't have `/*/usr/local/<folder-name>` directory. So I'd change it to something that would work in Windows.

Comment (hthillman) [reply docs/installation/install-livepeer/binary-release.md:54]: @leszko what's an appropriate filepath for Windows? I'm more of a Mac guy so I'm not sure what to suggest here - do you have windows experience?

Comment (leszko) [reply docs/installation/install-livepeer/binary-release.md:54]: For example `C:\Users\UserName\livepeer-folder`.

Comment (hnjolles1) [reply docs/installation/install-livepeer/binary-release.md:54]: That works, i will put that in the text instead of the line.

Comment (hnjolles1) [reply docs/installation/install-livepeer/binary-release.md:54]: @leszko i removed the .tar instructions -- we only use .zip for windows (unless there is a reason it should be there...)
Also, btw is the tar command for extracting on linux and darwin correct?
My research says  `tar -xzvf myfile.tar.gz` and our docs say `tar -zxvf livepeer-darwin-amd64.tar.gz`

Comment (leszko) [reply docs/installation/install-livepeer/binary-release.md]: > 

It's the same. The parameter `-xzvf` is the same as `-zxvf`

**Review by hthillman — APPROVED** [review]

File: docs/installation/install-livepeer/binary-release.md (hunk 2 above)
Comment (hnjolles1) [reply docs/installation/install-livepeer/binary-release.md]: okay, we'll just leave it as is

**Review by leszko — APPROVED** [review]

**Pull request merged**